src/Bicep.Local.Deploy/Extensibility/NestedDeploymentBuiltInLocalExtension.cs

```diff
@@ -47,7 +54,50 @@ public override async Task<LocalExtensibilityOperationResponse> CreateOrUpdate(R
                         ["parameters"] = request.Properties["parameters"]?.DeepClone(),
                     };
 
-                    var result = await LocalDeployment.Deploy(extensibilityHostManager, template, parameters.ToJsonString(), cancellationToken);
+                    var scopeProp = request.Properties["scope"]?.GetValue<string>();
+                    var subscriptionId = request.Properties["subscriptionId"]?.GetValue<string>();
+                    var resourceGroupName = request.Properties["resourceGroup"]?.GetValue<string>();
+
+                    LocalDeployment.Result result;
+                    if (scopeProp != null ||
+                        subscriptionId != null ||
+                        resourceGroupName != null)
+                    {
+                        try
+                        {
+                            var deploymentName = request.Properties["name"]!.GetValue<string>();
+                            var sourceUri = request.Properties["sourceUri"]!.GetValue<string>();
+                            if (sourceUri is null)
+                            {
+                                throw new ArgumentException("Failed to parse sourceUri.");
+                            }
+
+                            // TODO handle tenant & mg scope
+                            if (subscriptionId == null)
+                            {
+                                throw new NotImplementedException("Above-subscription scope is not supported yet.");
+                            }
+
+                            // TODO improve on this
+                            DeploymentLocator locator = new(null, null, subscriptionId, resourceGroupName, deploymentName);
+                            var configuration = configurationManager.GetConfiguration(new Uri(sourceUri));
+
+                            result = await LocalAzureDeployment.Deploy(configuration, credentialFactory, locator, template, request.Properties["parameters"]?.ToString() ?? "{}", cancellationToken);
```

Comment:
This is probably out of scope for this PR, but we may need to adopt the async pattern by setting status to `Running`, returning immediately, and handling polling separately, rather than waiting for the deployment to complete before returning the result. The `DeploymentExtensibleResourceJob` will time out if the nested deployment takes longer than one minute.